core/storage/job/gae_models.py

```diff
@@ -36,11 +36,40 @@
 
 
 class JobModel(base_models.BaseModel):
-    """Class representing a datastore entity for a long-running job."""
+    """Class representing a datastore entity for a long-running job.
+
+    Attributes:
```

Comment:
Hm, I don't think we need to repeat these, actually. Could you follow https://github.com/oppia/oppia/pull/2610 as a worked example?